psp/tests/models/test_load_models.py

```diff
@@ -5,28 +5,12 @@
 
 from psp.scripts.train_model import main as train_model_main
 from psp.serialization import load_model
-from psp.testing import make_test_nwp_data_source, run_click_command
+from psp.testing import run_click_command
 from psp.typings import X
 
-
-@pytest.fixture
-def nwp_data_source():
-    return make_test_nwp_data_source()
-
-
 # We ran the test once to get the results and pasted here the results.
 # This way we can make sure the output of previous models doesn't change.
-EXPECTED_OUTPUT = {
-    "model_v1": [1.0220418, 1.03555466, 1.04443434, 1.04860107, 1.04799241],
```

Comment:
This PR breaks the old serialized model but it doesn't matter too much. Actually they are probably fine but the data has changed.

Reply:
yea, thats ok to delete them. Just a bit of a tidy up! thanks